docs/02-architecture/06-client-side-architecture.md

```diff
@@ -43,6 +43,16 @@ See below for quick descriptions.
         - Omniture
         - Comscore
 
+### Polyfill
+
+We use [polyfill.io](https://polyfill.io) to polyfill many things, the selected polyfills are set in the [polyfill.io file](https://github.com/guardian/frontend/blob/master/static/src/javascripts/polyfill.io).
+
+We use aliases to refer to groups of polyfills. This seems to have disappeared from their documentation, so a [list of aliases](https://gist.github.com/AWare/c03ea6a6737bc890dd2aba0f2de068ea) has been compiled. This was generated by calling `listAliases` on [polyfill-library](https://github.com/Financial-Times/polyfill-library/).
+
+## Compilation
+
+We compile using the webpack babel loaded and the present-env preset. No settings or custom targets are loaded for this.
+
```

Comment:
Shall we make explicit that the [`browserslist` default targets](https://github.com/browserslist/browserslist#queries) are used?